openassessment/assessment/test/test_staff.py

```diff
@@ -78,8 +78,10 @@ def test_create_assessment_not_required(self, key):
         self.assertEqual(assessment["points_earned"], OPTIONS_SELECTED_DICT[key]["expected_points"])
         self.assertEqual(assessment["points_possible"], RUBRIC_POSSIBLE_POINTS)
 
-        # ensure submission is marked as finished
+        # Ensure submission and workflow are marked as finished
         self.assertTrue(staff_api.assessment_is_finished(tim_sub["uuid"], self.STEP_REQUIREMENTS))
```

Comment:
Nit: I'd consider a helper method here for these 3 lines, since I assume everyplace we check assessment_is_finished we should also check the workflow status.